frontends/ol-components/src/components/ThemeProvider/typography.ts

```diff
@@ -141,6 +141,7 @@ const globalSettings: ThemeOptions["typography"] = {
     fontStyle: "normal",
     fontSize: pxToRem(14),
     lineHeight: pxToRem(14),
+    textTransform: "none",
```

Comment:
In https://github.com/mitodl/mit-open/pull/889/files#diff-2cc4a93727691551d62dc05f2ed43bc01fbe2a7a632861ab97e7b72471408037R59 I added `...typography.button` to our medium button. `typography.button` inherited an extra `text-transform: uppercase` from MUI, which also has a `typography.button`. **That was the only inherited style**. And now it's fixed.

We're using this, rather than `typography.buttonMedium` or something because we want MUI buttons to look good, too. MUI uses their own button for things like tabs and pagination controls.